docs/src/whatsnew/latest.rst

```diff
@@ -79,6 +79,9 @@ This document explains the changes made to Iris for this release
    working properly. (Main pull request: :pull:`5437`, more detail:
    :pull:`5430`, :pull:`5431`, :pull:`5432`, :pull:`5434`, :pull:`5436`)
 
+#. `@acchamber` removed several warnings from iris related to Numpy 1.25 deprecations
```

Comment:
```suggestion
#. `@acchamber`_ removed several warnings from iris related to Numpy 1.25 deprecations
```

Reply:
Since you don't seem to be in the "common_links" file either, you will have to add in a link to your name, as on line 90.